cln/cln_client.go

```diff
@@ -65,7 +66,7 @@ func (c *ClnClient) IsConnected(destination []byte) (bool, error) {
 	}
 
 	for _, peer := range peers {
-		if pubKey == peer.Id {
+		if pubKey == peer.Id && peer.Connected {
```

Comment:
Not related as it is not introduced in this PR so just asking why don't we populate the peerID to the ListPeers endpoint? https://lightning.readthedocs.io/lightning-listpeers.7.html

Reply:
Good question. I don't think there's a good reason. Probably because glightning returns an error if the peer is not found.

Reply:
Fixed